lib/footer.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+
+class Footer extends StatelessWidget {
+  const Footer({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final TextStyle? footerTextStyle =
+        Theme.of(context).textTheme.bodyMedium?.apply(color: Colors.white);
+
+    return ColoredBox(
+      color: Theme.of(context).colorScheme.secondary,
+      child: const Padding(
+        padding: EdgeInsets.all(40.0),
+        child: Column(
+          children: [
+            ContactDetail(icon: Icons.phone, text: '06 235 873 74'),
+            ContactDetail(icon: Icons.phone, text: '0592 769 056'),
+            ContactDetail(
+                icon: Icons.mail, text: 'info@redactiebureaumaritaweener.nl'),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class ContactDetail extends StatelessWidget {
+  const ContactDetail({
+    super.key,
+    required this.icon,
+    required this.text,
+  });
```

Comment:
```suggestion
class _ContactDetail extends StatelessWidget {
  const _ContactDetail({
    required this.icon,
    required this.text,
    super.key,
  });
```
I would prefer this to be a private widget since we are only going to use it in this file.
`super` parameters are always last, `required` ones are always first. :)